terraform/database.tf

```diff
@@ -418,7 +418,7 @@ resource "google_cloud_scheduler_job" "backup-database-worker" {
         fileType  = "SQL"
         uri       = "gs://${google_storage_bucket.backups.name}/database/${google_sql_database.db.name}",
         databases = [google_sql_database.db.name]
-        offload   = true,
+        offload   = false,
```

Comment:
I'm not sure this is a good idea unless we _really_ bump the timeout. It can take 15+min

Reply:
The timeout now is 30 mins. This is the best way to have the retry logic work without relying on external services. (Cloud SQL or GCS) to verify the backup.

Reply:
Also, 30mins is the max attempt_deadline allowed: https://cloud.google.com/scheduler/docs/reference/rpc/google.cloud.scheduler.v1#:~:text=The%20failed%20attempt%20can%20be,15%20seconds%20and%2024%20hours.